components/avatar/avatar.css

```diff
@@ -0,0 +1,101 @@
+@import url(/landing.css);
+@import url(/components/instalation/instalation.css);
+
+
+body{
+    display: flex;
+    max-width: 100%;
+    flex-wrap: wrap;
+}
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    
+}
+.avatar_img{
+    border-radius: 50%;
+    
+}
+.large_img{
+    width: 9rem;
+    height: 9rem;
+}
+.medium_img{
+    width: 7rem;
+    height: 7rem;
+
+}
+.small_img{
+    width: 3.5rem;
+    height: 3.5rem;
+}
+.xs_img{
+    width: 2rem;
+    height: 2rem;
+}
+span{
+    color: var(--primary-color);
+}
+.component_page{
+    display: flex;
+    flex-direction: column;
+    gap: 1.5rem;
+    padding:4rem;
+    width: 100%;
+}
+.all_avatar_img, .text_avatar{
+    border: 1px solid 
+    #94a3b8;
```

Comment:
You could combine these two lines to have it in a single line. e.g. `border: 1px solid #94a3b8`